scripts/travis_script.sh

```diff
@@ -7,15 +7,17 @@ set -ex
 export DISPLAY=:99.0
 sh -e /etc/init.d/xvfb start || true
 
-# Install in-place and enable the server extension
 export PATH="$HOME/miniconda/bin:$PATH"
-pip install -v .
-jupyter serverextension enable --py jupyterlab
 
 npm run clean
 npm run build
-npm test
-npm run test:coverage
+npm test || npm test
+npm run test:coverage || npm run test:coverage
```

Comment:
IIUC it is a race condition on the server that only happens on Travis.  The notebook itself runs its tests 3 times: https://github.com/jupyter/notebook/blob/f0352f6ebca08ae996deebaa1fbd57c013f806b9/.travis.yml#L37